app/api/api_v1/endpoints/rule.py

```diff
@@ -0,0 +1,139 @@
+from fastapi import APIRouter, HTTPException, Depends
+from requests import Session
+
+import crud
+from api import deps
+from schemas import Message, RulesDB, RuleDB, CreateRule, CreateCondition
+
+router = APIRouter()
+
+
+@router.post("/", response_model=RuleDB)
+def create_rule(
+        rule: CreateRule,
+        db: Session = Depends(deps.get_db)
```

Comment:
probable this is not `requests.Session` but `sqlalchemy.orm.Session`

Reply:
Yes, changed it to sqlalchemy.orm.Session